content/pt-br/docs/reference/issues-security/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Problemas e Segurança do Kubernetes
+weight: 40
+toc-hide: true
```

Comment:
This does not exist in the original version. It should not be added in the translation.